massa-execution-worker/src/execution.rs

```diff
@@ -1021,7 +1042,7 @@ impl ExecutionState {
         let mut execution_context = ExecutionContext::active_slot(
             self.config.clone(),
             *slot,
-            exec_target.as_ref().map(|(b_id, _)| *b_id),
+            exec_target.map(|i| i.0), // .as_ref().map(|(b_id, _)| *b_id),
```

Comment:
why changing to this less readable/explicit alternative ?

Reply:
Fixed